test/postman_test/BCDA_Tests_Sequential.postman_collection.json

```diff
@@ -1351,7 +1351,9 @@
 					"script": {
 						"id": "b822f300-2ead-4f25-b623-22c68df6b017",
 						"exec": [
-							"let timestamp = new Date().toJSON();",
+							"let date = new Date()",
```

Comment:
Looks like this is NOT the problem.  This can probably be reverted.  The issue is with the `Z` in the timestamp.